src/index.html

```diff
@@ -21,7 +24,406 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo"
+        href="#"
+      >
+        <img
+          class="header__img"
+          src="./images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="cards-container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="iMac images"
+          />
+
+          <div class="card__info">
+            <h1 class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
```

Comment:
This tag used for title of the full page, so consider will be better if you'll use the another tag, fix it everywhere
```suggestion
            <h3 class="card__name">
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
            </h3>
```